.github/workflows/build.yml

```diff
@@ -18,13 +18,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
```

Comment:
Looks like it's using actions/checkout@v2 and v3, and actions/setup-go@v2 and v3. Is that intentional?

Reply:
Nope, locally it gave me a merge conflict. So not quite sure how this happened. Resolved it though!